rust/worker/src/types/metadata.rs

```diff
@@ -280,6 +280,75 @@ pub(crate) fn update_metdata_to_metdata(
     Ok(metadata)
 }
 
+/*
+===========================================
+Metadata queries
+===========================================
+*/
+
+pub(crate) enum Where {
```

Comment:
do we need to add the conversion for these from proto?

Reply:
Brain fart. Done :)